rest/model/src/main/java/io/syndesis/model/extension/ExtensionDetail.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.syndesis.model.extension;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import io.syndesis.model.ResourceIdentifier;
+import org.immutables.value.Value;
+
+import java.io.Serializable;
+import java.util.Set;
+
+@Value.Immutable
+@JsonDeserialize(builder = ExtensionDetail.Builder.class)
+public interface ExtensionDetail extends Serializable {
+
+    Extension getExtension();
+
+    Set<ResourceIdentifier> activeIntegrations();
```

Comment:
An extension may not have any active integration, should't we have a default returning an empty set ?

Reply:
Yep, also the method signature should be `getActiveIntegrations()`